docs/source/api/pystiche_papers.ulyanov_et_al_2016.rst

```diff
@@ -1,6 +1,95 @@
 ``pystiche_papers.ulyanov_et_al_2016``
 =========================================
 
+Unfortunately, the parameters in the reference implementation differ from the parameters
+described in the paper. If ``impl_params is True``, the parameters from the reference
+implementation are used instead of the parameters from the paper. The following
+functions are affected by the parameter ``impl_params``:
+
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.training`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.stylization`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.optimizer`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.lr_scheduler`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.content_transform`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.style_transform`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.batch_sampler`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.content_loss`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.style_loss`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.transformer`.
+
+
+.. _table-branches-ulyanov_et_al_2016:
+
+The original authors have provided two implementations. Unfortunately, the
```

Comment:
IIRC the authors published two papers: one without and a later one with instance normalization. If that is the case, we should describe this properly. Although they live in the same repo, each branch is an independent reference implementation for a paper.